### PR TITLE
Check if project token exist

### PR DESCRIPTION
**Pull request by kkatusic**
Resolve: #73

### CODE REVIEW

File: src/app/actions/tokenExist.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export async function tokenExist(param: { userAddress: string }) {
+  const { userAddress } = param;
+
+  const db: Db = await getMongoDB();
+  const projectCollection: Collection = db.collection("project");
+
+  const normalizedWalletAddress = userAddress.toLowerCase();
+
+  console.log({ normalizedWalletAddress });
```
Comment (aminlatifi) [inline src/app/actions/tokenExist.ts:14]: Remove this please

**Review by aminlatifi — APPROVED** [review]
The query is right @kkatusic 
Thanks

File: src/app/actions/tokenExist.ts (hunk 1 above)
Comment (kkatusic) [reply src/app/actions/tokenExist.ts]: removed

Comment (kkatusic) [reply src/app/actions/tokenExist.ts]: removed, thx

**Pull request merged**